components/button/style.scss

```diff
@@ -71,8 +72,8 @@
 			0 1px 1px color( theme( button ) shade( 30% ) ),
 			-1px 0 1px color( theme( button ) shade( 30% ) );
 
-		&:hover,
-		&:focus:not(:disabled):not([aria-disabled="true"]) {
+			&:hover,
+			&:focus:not(:disabled):not([aria-disabled="true"]) {
```

Comment:
This is probably an unwanted change right?

Reply:
Yes indeed! Fix pushed.